python/pyspark/sql/avro/functions.py

```diff
@@ -73,7 +73,7 @@ def from_avro(
     [Row(value=Row(avro=Row(age=2, name='Alice')))]
     """
 
-    sc = SparkContext._active_spark_context  # type: ignore[attr-defined]
+    sc = cast(SparkContext, SparkContext._active_spark_context)  # type: ignore[attr-defined]
```

Comment:
To not to repeat this every time ‒ we should be able to drop `type: ignore[attr-defined]` from `SparkContext._active_spark_context`